toolkit/templates/forms/pricing.html

```diff
@@ -30,6 +32,10 @@
     <span class="validation-message">
       {{ error }}
     </span>
+    {% elif message %}
+    <span class="message-message">
```

Comment:
There is also prior art... cf. `temporary-message-message` in `_temporary_message.scss`